pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/Designer.java

```diff
@@ -83,11 +110,22 @@ public void start(Stage stage) throws IOException {
         Parent root = loader.load();
         Scene scene = new Scene(root);
 
-        stage.setTitle("PMD Rule Designer (v " + PMDVersion.VERSION + ')');
-        setIcons(stage);
-
         stage.setScene(scene);
+
+        if (owner.isDeveloperMode()) {
+            // only close after initialization succeeded.
+            // but before stage.show to reduce unwanted noise
+            System.err.close();
```

Comment:
wasn't this behavior the other way around before? we closed `System.err` if **not** using `-v` / `--verbose`?

Reply:
Lol you're right

Reply:
Thanks for catching that

Reply:
Fixed in 8d7b885d83